### DISCUSSION TITLE
Can't load photos and can't edit persons after 1.115

### DISCUSSION THREAD

**Discussion by XxJooO** [immich-app/immich#13031] [Q&A]
### The bug

I've updated the version from v1.114 to 1.115. Up to 1.114 it was fine. This was the first time that the gui could not show all photos "Fehler beim Laden des Bildes". Same behavior when showing persons "Fehler beim Laden des Bildes"

And another problem: I can't edit people directly in the photo. As I remember I could that before?! When I click on the minus, I can delete recognition of this person. I can't click on the other symbol and I can only edit names of person when I go to Erkunden-Personen-Alles anzeigen

### The OS that Immich Server is running on

docker in proxmox Container Linux 6.8.12-pve

### Version of Immich Server

1.116.2

### Version of Immich Mobile App

1.115.0 build 159

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

#
# WARNING: Make sure to use the docker-compose.yml of the current release:
#
# https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
#
# The compose file on main may not be compatible with the latest release.
#

name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends:
    #   file: hwaccel.transcoding.yml
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: docker.io/redis:6.2-alpine@sha256:328fe6a5822256d065debb36617a8169dbfbd77b797c525288e465f56c1d392b
    healthcheck:
      test: redis-cli ping || exit 1
    restart: always

  database:
    container_name: immich_postgres
    image: docker.io/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: '--data-checksums'
    volumes:
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    healthcheck:
      test: pg_isready --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' || exit 1; Chksum="$$(psql --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}'>
      interval: 5m
      start_interval: 30s
      start_period: 5m
    command: ["postgres", "-c" ,"shared_preload_libraries=vectors.so", "-c", 'search_path="$$user", public, vectors', "-c", "logging_collector=on", "-c", "max_wal_size>
    restart: always

volumes:
  model-cache:

### Reproduction steps

1. Installed via docker compose
2.
3.
...

### Additional information

![image](https://github.com/user-attachments/assets/68751e3d-f930-42a6-b432-8f23279eee71)

**Comment by bo0tzz** [comment]
Check the running containers on your system and make sure there isn't an old microservices one sitting around.

**Comment by XxJooO** [comment]
One more hint about this???

I did every update as a user with sudo docker compose pull && sudo docker compose up -d

**Comment by XxJooO** [comment]
I did a completely new setup and re-imported all of my photos through the web-Interface. So I made sure to have completely new databases. So far there is no change in behaviour. There are still photos that can't be loaded in the first way, when I refresh the browser then they are loaded. 
And the problem with the faces is still there. When a photo is shown, I can only add a name of a face when still one is given on the photo. If no one is given, I can only see what is shown above...
The machine in proxmox is set up with Ubuntu 24.04, 12Gigs of RAM, 4 cores and 30Gigs of SSD.

So I think there is still a problem. Is there anything I can do.

**Comment by umrath** [comment]
With 0.117.0 I have the same phenomen: Some picture won't load.
(The server is brand new, just started to import today.)

Edit: So far it seems to affect PNG-objects only. But cannot fully rule out others because I didn't check all files.

**Reply by alextran1502** [reply]
Hi @umrath, can you describe the buggy behavior and see if you can provide reproducible steps? But let's make sure all the jobs have finished

**Reply by umrath** [reply]
Actually, it looks like a temporary glitch, maybe caused by the server being overloaded.
I'm currently uploading quite a bit of objects and I cannot reproduce the problem anymore. The formerly faulty picture are working now.
